tests/Kernel/ContactFormRenderTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel;
+
+use Drupal\contact\Entity\ContactForm;
+use Drupal\Tests\oe_contact_forms\Kernel\ContactFormTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests that our contact forms renders with ecl markup.
+ */
+class ContactFormRenderTest extends ContactFormTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'ui_patterns',
+    'ui_patterns_library',
+    'oe_theme_helper',
+    'breakpoint',
+    'responsive_image',
+    'image',
+    'oe_theme_contact_forms',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installConfig(['system', 'image', 'responsive_image', 'contact']);
+    $this->container->get('theme_installer')->install(['oe_theme']);
+    $this->container->get('theme_handler')->setDefault('oe_theme');
+    $this->container->set('theme.registry', NULL);
+  }
+
+  /**
+   * Tests that corporate contact form is rendered with the correct ECL markup.
+   */
+  public function testContactForm(): void {
+    $contact_form = ContactForm::create(['id' => 'oe_contact_form']);
+    $contact_form->setThirdPartySetting('oe_contact_forms', 'is_corporate_form', TRUE);
+    $header = 'this is a test header';
+    $contact_form->setThirdPartySetting('oe_contact_forms', 'header', $header);
```

Comment:
You use the $header variable only on this line, let's not set useless variables, can't we just set the value right away?